### DISCUSSION TITLE
Whitelist creation of Projects for specific users

### DISCUSSION THREAD

**Discussion by Fabian9799** [appwrite/appwrite#1423] [Ideas]
I think it would be a good idea to have a .env variable to specify specific users that are able to create projects.

At the moment everyone that is added to a team can also kick everyone out of it. That makes it a little bit complicated to view all projects that are in a appwrite instance if someone kicks a person out of a project

maybe something like `_APP_CONSOLE_PROJECT_CREATE_EMAILS`

It would probably also help to give the "admin" account access to all projects or limit the permissions on who can kick people out of projects

**Comment by eldadfux** [comment]
We do plan you add more access control with different member roles, so regular members won't be able to remove an admin, will this help?

**Reply by Fabian9799** [reply]
Yes this would definetly help with keeping everything more managed and organized
